packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.Props.ts

```diff
@@ -18,7 +18,7 @@ export interface IDropdownProps extends ISelectableDroppableTextProps<HTMLDivEle
   /**
    * Callback issues when the selected option changes
    */
-  onChanged?: (option: IDropdownOption, index?: number) => void;

```

Comment:
I had put this as a major change so that it would get pushed into the 5.0 branch. That being said, I thought the deprecated intermediate step could be bypassed if it was tied to a major release. Regardless, I'll close this pull request and let the param/naming update handle this.